neqo-crypto/src/agent.rs

```diff
@@ -122,7 +124,7 @@ macro_rules! preinfo_arg {
         pub fn $v(&self) -> Option<$t> {
             match self.info.valuesSet & ssl::$m {
                 0 => None,
-                _ => Some(<$t>::try_from(self.info.$f).unwrap()),
+                _ => Some(<$t>::try_from(self.info.$f).ok()?),
```

Comment:
Not familiar with the code. Silently drops the `None` case. How about adding a `qdebug!` via [`Result::inspect_err`](https://doc.rust-lang.org/std/result/enum.Result.html#method.inspect_err)?